config/environments/test.rb

```diff
@@ -39,4 +39,20 @@
 
   # Raises error for missing translations
   # config.action_view.raise_on_missing_translations = true
+  # Setup mailer
+
+  config.action_mailer.default_url_options = { host: 'localhost:3000' }
```

Comment:
@jsyeo @lowjoel  Is there any gems which we are going to use to handle secrets ?